kola/tests/ostree/basic.go

```diff
@@ -0,0 +1,243 @@
+// Copyright 2018 Red Hat, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ostree
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+	"time"
+
+	"github.com/coreos/mantle/kola/cluster"
+	"github.com/coreos/mantle/kola/register"
+	"github.com/coreos/mantle/kola/tests/rpmostree"
+	"github.com/coreos/mantle/platform"
+)
+
+func init() {
+	register.Register(&register.Test{
+		Run:         ostreeBasicTest,
+		ClusterSize: 1,
+		Name:        "rhcos.ostree.basic",
```

Comment:
The test names (and the test names in `ostree/unlock.go`) should be changed to conform with the [kola test namespacing policy](https://github.com/coreos/mantle/blob/master/README.md#kola-test-namespacing) to either `coreos.ostree.*` or `ostree.*` as they support multiple distributions (the `rhcos.*` top level namespace would be if the tests only supported `rhcos`)

Reply:
Makes sense.  Should I migrate the `kola/tests/rpmostree` tests similarly?

Reply:
Yeah, if I missed any of them feel free to add a commit or a separate PR.